### PR TITLE
Docs for Istio and IPv6 support

### PR DESCRIPTION
**Pull request by aaha97**

### CODE REVIEW

File: docs/istio_support.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# AKO on Istio
```
Comment (apalsule) [inline docs/istio_support.md:1]: please mention that this feature is in tech preview in this release

File: docs/crds/avinfrasetting.md [hunk 2]
```diff
@@ -167,3 +168,12 @@ For the subset of ingresses, that refer to an ingress class which in turn refers
 For passthrough routes/ingresses, setting `l7Settings:shardSize` present in AviInfrasetting CRD overrides setting `L7Settings.passthroughShardSize` present in values.yaml. <br>
 **Note**:  Value `DEDICATED` is not supported when AviInfrasetting CRD is applied to the passthrough route/ingress.
 
+#### Configure IPv6
+
+AviInfraSetting CRD can be used to enable IPv6, IPv4 or both IPv4 and IPv6 vips on virtualservices created by AKO.
```
Comment (apalsule) [inline docs/crds/avinfrasetting.md:173]: Please mention that this feature is in techpreview stage in this release

Comment (aaha97) [reply docs/crds/avinfrasetting.md:173]: added tech preview to title

File: docs/istio_support.md (hunk 1 above)
Comment (aaha97) [reply docs/istio_support.md]: added tech preview to title

File: docs/values.md [hunk 3]
```diff
@@ -89,6 +89,18 @@ The `blockedNamespaceList` lists the Kubernetes/Openshift namespaces blocked by
...
 
+### AKOSetttings.istioEnabled (Tech Preview)
+
+AKO can be deployed in Istio environment. Setting this to `true` indicates to AKO that the environment is Istio. Default value is `false`.
+
+### AKOSetttings.ipFamily (Tech Preview)
+
+`V6` is currently supported only for `vCenter` cloud with `calico` CNI.
+
+AKO can be deployed with ipFamily as `V4` or `V6`. When ipFamily is set to `V6`, AKO looks for `V6` IP for nodes from calico annotation and creates routes on controller. Only servers with `V6` IP will get added to Pools.
```
Comment (swathinsankaran) [inline docs/values.md:100]: what will be the value if the user wants to use dual-stack?

**Comment by akshayhavile** [comment]
@aaha97 please rebase your branch

File: docs/values.md (hunk 3 above)
Comment (aaha97) [reply docs/values.md:100]: dual stack for backend is not supported

**Comment by swathinsankaran** [comment]
build ako

**Pull request merged**